dbms/src/Common/MemoryTracker.h

```diff
@@ -38,6 +38,9 @@ class MemoryTracker
     /// To test exception safety of calling code, memory tracker throws an exception on each memory allocation with specified probability.
     double fault_probability = 0;
 
+    /// To test the accuracy of memory track, it throws an exception when the part exceeding the tracked amount is greater than accuracy_diff.
+    Int64 accuracy_diff = 0;
```

Comment:
how about rename it into `accuracy_diff_for_test`

Reply:
Good idea. I named it without the identification of test just because ```fault_probability``` is also for test without the identification. Anyway, this variable name is confusing and I'll modify it.